### PR TITLE
GPU: Added some display transfer tests

### PR DESCRIPTION
**Pull request by Subv**
Citra doesn't yet pass all of these.

### CODE REVIEW

**Comment by archshift** [comment]
LGTM. @yuriks, @lioncash?

**Comment by Subv** [comment]
With the latest master, Citra is now only failing the "scaled_blending" test

File: source/tests/gpu/displaytransfer.cpp [hunk 1]
```diff
@@ -359,29 +531,167 @@ static bool RGBA8_To_RGBA8_Scaled_Blending(u32* input, u32* output) {
...
     Test(tag, "RGBA8_To_RGBA8", RGBA8_To_RGBA8(input, output), true);
     Test(tag, "RGBA8_To_RGB8", RGBA8_To_RGB8(input, output), true);
     Test(tag, "RGBA8_To_RGB565", RGBA8_To_RGB565(input, output), true);
     Test(tag, "RGBA8_To_RGB5A1", RGBA8_To_RGB5A1(input, output), true);
     Test(tag, "RGBA8_To_RGBA4", RGBA8_To_RGBA4(input, output), true);
+    // RGB8 -> X tests
+    Test(tag, "RGB8_To_RGB8", RGB8_To_RGB8(input, output), true);
+    //Test(tag, "RGB8_To_RGB565", RGB8_To_RGB565(input, output), true); Freezes
+    //Test(tag, "RGB8_To_RGB5A1", RGB8_To_RGB5A1(input, output), true); Freezes
+    // Test(tag, "RGB8_To_RGBA4", RGB8_To_RGBA4(input, output), true);
```
Comment (lioncash) [inline source/tests/gpu/displaytransfer.cpp]: Why are these ones commented out?

**Comment by Subv** [comment]
Updated to include one more test

**Comment by bunnei** [comment]
LGTM

**Pull request merged**